gems/pending/util/postgres_admin.rb

```diff
@@ -53,6 +53,20 @@ def self.database_disk_filesystem
     "xfs".freeze
   end
 
+  def self.initialized?
+    !Dir[data_directory.join("*")].empty?
+  end
+
+  def self.service_running?
+    LinuxAdmin::Service.new(service_name).running?
+  end
+
+  def self.local_server_status
+    return "running" if service_running?
+    return "initialized and stopped" if initialized?
+    "not initialized"
```

Comment:
Sounds good. I may add something about being in recovery (if the server is acting as a standby database) so I'll likely break this all out into a separate class then.